ui/core/src/ui-core/slider/Slider.tsx

```diff
@@ -289,8 +304,8 @@ function TooltipTrack(props: TooltipTrackProps) {
 
   let tooltipText = "";
   if (multipleValues) {
-    const sourceValue = formatTooltip ? formatTooltip(source.value) : source.value.toString();
-    const targetValue = formatTooltip ? formatTooltip(target.value) : target.value.toString();
+    const sourceValue = formatTooltip ? formatTooltip(source.value) : /* istanbul ignore next */ source.value.toString();

```

Comment:
It is also now possible to make `formatTooltip` prop required and remove these ignores.

Reply:
Good catch.  I didn't even notice that this component is internal until you made the suggestion.